src/orion/analysis/base.py

```diff
@@ -26,7 +28,8 @@
 }
 
 
-def average(trials, group_by="order", key="best", return_var=False):
+# pylint:disable=dangerous-default-value
+def average(trials, group_by="order", keys=["best"], return_var=False):
```

Comment:
I think the best would be to still support a string, and convert to list if string.